src/serac/physics/solid.cpp

```diff
@@ -122,6 +124,27 @@ Solid::Solid(std::shared_ptr<mfem::ParMesh> mesh, const Solid::InputOptions& opt
   }
 }
 
+Solid::~Solid()
+{
+  // Update the mesh with the new deformed nodes if requested
+  if (keep_deformation_after_destructor_) {
+    reference_nodes_->Add(1.0, displacement_.gridFunc());
```

Comment:
Minor: `*reference_nodes_ += displacement_.gridFunc();`?

Reply:
I keep forgetting about the expression templates!